pkg/cmd/openshift-controller-manager/controller/build.go

```diff
@@ -12,19 +17,32 @@ import (
 
 // RunController starts the build sync loop for builds and buildConfig processing.
 func RunBuildController(ctx *ControllerContext) (bool, error) {
+
 	imageTemplate := variable.NewDefaultImageTemplate()
 	imageTemplate.Format = ctx.OpenshiftControllerConfig.Build.ImageTemplateFormat.Format
 	imageTemplate.Latest = ctx.OpenshiftControllerConfig.Build.ImageTemplateFormat.Latest
 
-	buildClient := ctx.ClientBuilder.OpenshiftBuildClientOrDie(bootstrappolicy.InfraBuildControllerServiceAccountName)
-	externalKubeClient := ctx.ClientBuilder.ClientOrDie(bootstrappolicy.InfraBuildControllerServiceAccountName)
+	cfg := ctx.ClientBuilder.ConfigOrDie(bootstrappolicy.InfraBuildControllerServiceAccountName)
+	cfg.QPS = cfg.QPS * 2
+	cfg.Burst = cfg.Burst * 2
```

Comment:
I don't think you should be setting these, we don't do that anywhere else.

Reply:
The default settings are insufficient for the build controller's requirements and it is inhibiting the ability of the system to scale under high incoming build load.  We can scale up api servers, we cannot scale up controllers, thus we need to set the client rate limit higher.

If you have another way you think we should set it, please advise, if you are simply saying we should not be allowed to set our client limits higher, the scale/perf test results indicate otherwise.  We are getting throttled hard.

Reply:
Thanks, I was curious of reasons behind these. Maybe worth adding  comment there, why we need those higher than standard.